### PR TITLE
Display different message if all vouchers are used (Fixes: #37)

### PR DESCRIPTION
**Pull request by doobry-systemli**

### CODE REVIEW

File: default_translations/de/messages.de.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ start:
 index:
   title: Verwalte deinen E-Mail Account
   voucher-headline: Deine Einladungscodes
+  voucher-limit: Du hast keine Vouchers mehr verfügbar.
```
Comment (0x46616c6b) [inline default_translations/de/messages.de.yml]: We should use the term `Einladungscodes` like in the headline.

File: default_translations/en/messages.en.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ start:
 index:
   title: Manage your email-account
   voucher-headline: Your invite codes
+  voucher-limit: You don't have any vouchers left.
```
Comment (0x46616c6b) [inline default_translations/en/messages.en.yml]: We should use the term `invite code` like in the headline.

**Comment by t2d** [comment]
@0x46616c6b good catch! I changed these and even more translations. Merge?

**Comment by doobry-systemli** [comment]
Yep, well spotted. Should be merged.

**Comment by doobry-systemli** [comment]
.. once the behat tests are fixed ;)

**Pull request merged**